pos_supplierinfo_search/models/product.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    # technical field used in POS frontend
+    supplier_data_json = fields.Char(
+        "Supplier data dict", readonly=True, compute="_compute_supplier_data_json"
+    )
```

Comment:
```suggestion
    supplier_data_json = fields.Char(
        "Supplier data dict",
        help="Technical field: Used in POS frontend to search products by supplierinfo",
        compute="_compute_supplier_data_json",
    )
```

- Readonly is already True by default for computed fields
- Move comment to field help